slasher/db/min_max-span_test.go

```diff
@@ -0,0 +1,118 @@
+package db
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"testing"
+
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+)
+
+type spanMapTestStruct struct {
+	validatorID uint64
```

Comment:
Is this the index or an ID? Can we call validatorIdx if it is the index in the state.Validator registry?

Reply:
changed all